features/datum.feature

```diff
@@ -75,150 +71,157 @@ Functionaliteit: leveren van een datum
-      | datum.langFormaat | onbekend      |
+      Gegeven de persoon met burgerservicenummer '000000139' heeft de volgende 'overlijden' gegevens
+        | naam                     | waarde   |
+        | datum overlijden (08.10) | 00000000 |
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                          |
+        | type                | RaadpleegMetBurgerservicenummer |
+        | burgerservicenummer | 000000139                       |
+        | fields              | overlijden.datum                |
+        Dan heeft de response een persoon met de volgende 'overlijden' gegevens
+        | naam              | waarde        |
+        | datum.type        | DatumOnbekend |
+        | datum.onbekend    | true          |
+        | datum.langFormaat | onbekend      |
 
     Abstract Scenario: <type> in <groep> <veld>
-      Gegeven het systeem heeft een persoon met de volgende gegevens
-      | naam                | waarde    |
-      | burgerservicenummer | 555550001 |
-      En de persoon heeft de volgende '<groep>' gegevens
-      | naam   | waarde     |
-      | <veld> | <GbaDatum> |
-      Als personen wordt gezocht met de volgende parameters
-      | naam                | waarde                          |
-      | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550001                       |
-      | fields              | <groep>.<veld>                  |
-      Dan heeft de response een persoon met de volgende '<groep>' gegevens
-      | naam               | waarde        |
-      | <veld>.type        | <type>        |
-      | <veld>.datum       | <datum>       |
-      | <veld>.jaar        | <jaar>        |
-      | <veld>.maand       | <maand>       |
-      | <veld>.onbekend    | <onbekend>    |
-      | <veld>.langFormaat | <langFormaat> |
+      Gegeven de persoon met burgerservicenummer '000000140' heeft de volgende gegevens
+        | naam                  | waarde     |
+        | geboortedatum (03.10) | <GbaDatum> |
+        Als personen wordt gezocht met de volgende parameters
+        | naam                | waarde                          |
+        | type                | RaadpleegMetBurgerservicenummer |
+        | burgerservicenummer | 000000140                       |
+        | fields              | <groep>.<veld>                  |
+        Dan heeft de response een persoon met de volgende '<groep>' gegevens
+        | naam               | waarde        |
+        | <veld>.type        | <type>        |
+        | <veld>.datum       | <datum>       |
+        | <veld>.jaar        | <jaar>        |
+        | <veld>.maand       | <maand>       |
+        | <veld>.onbekend    | <onbekend>    |
+        | <veld>.langFormaat | <langFormaat> |
 
       Voorbeelden:
       | groep                 | veld                      | type           | GbaDatum | datum      | jaar | maand | onbekend | langFormaat  |
```

Comment:
De eerste 2 kolommen in deze voorbeeld tabel zijn onnodig. De waardes zijn in alle vier de regels gelijk en dus kunnen ze ook gewoon opgenomen worden in het abstracte scenario als vaste waardes.

Reply:
Aangepast